R/snapshot-reporter.R

```diff
@@ -87,7 +90,7 @@ SnapshotReporter <- R6::R6Class("SnapshotReporter",
         self$cur_snaps[[self$test]] <- self$old_snaps[[self$test]]
 
         if (self$i > 0) {
-          testthat_warn("Snapshots reset after error/skip")
+          testthat_warn(paste0("Snapshots reset after error/skip: ", test))
```

Comment:
Won't the test name have been printed above?

Reply:
Probably, let's try and see.